src/tree/AttachedAccountsTreeItem.ts

```diff
@@ -56,6 +56,10 @@ export class AttachedAccountsTreeItem extends AzureParentTreeItem {
         this._loadPersistedAccountsTask = this.loadPersistedAccounts();
     }
 
+    public static isPlatformWindows(): boolean {
+        return process.platform === 'win32';
+    }
```

Comment:
It looks like this already exists here: https://github.com/microsoft/vscode-cosmosdb/blob/8d86a075ef090aec387c9a9c3861b42f1c383e8b/src/constants.ts#L6

Reply:
That's true. Line 41 can use the same value as well. Will update both lines.